### PR TITLE
Add Customers Report page

### PR DESCRIPTION
**Pull request by Aljullu**
Part of #918.

This PR creates the _Customers_ report page with a table connected to the SwaggerHub endpoint.

### Screenshots
image

### Detailed test instructions:
- Go to the _Customers_ report.
- Verify there is a table with the correct columns.
- Verify the date range filter is hidden.

### Follow-up work
- Load more data for each _Customer_ from the ` /wc/v3/customers` endpoint.
- Table summary.

### CODE REVIEW

File: client/analytics/report/customers/table.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+	}
+
+	getRowsContent( customers ) {
+		const { query } = this.props;
+		const currentInterval = getIntervalForQuery( query );
+		const formats = getDateFormatsForInterval( currentInterval );
+
+		return customers.map( customer => {
+			const {
+				adv,
```
Comment (psealock) [inline client/analytics/report/customers/table.js]: whats `adv`?

**Review by psealock — COMMENTED** [review]
Looking good so far, just a couple comments.

File: client/analytics/report/customers/table.js (hunk 1 above)
Comment (Aljullu) [reply client/analytics/report/customers/table.js]: Actually it's `average_order_value`.

**Comment by Aljullu** [comment]
@psealock thanks for the feedback. This should be ready for another review.

**Review by psealock — APPROVED** [review]

**Pull request merged**